src-os/ptime_clock_stubs.c

```diff
@@ -106,6 +110,45 @@ CAMLprim value ocaml_ptime_clock_now_d_ps (value unit)
   CAMLreturn (pair);
 }
 
+#elif defined(OCAML_PTIME_WIN)
+
+CAMLprim value ocaml_ptime_clock_now_d_ps (value unit)
+{
+  CAMLparam1 (unit);
+  CAMLlocal1 (pair);
+  long sec, usec;
+  SYSTEMTIME stime;
+  FILETIME ftime;
+  uint64_t time;
```

Comment:
IIRC `uint64_t` is Visual Studio 2010 onwards only. It would be more robust to use a `ULARGE_INTEGER` struct.

Reply:
@IndiscriminateCoding would please be possible for you to change the code as such.